app/controllers/application_controller.rb

```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  before_action :authenticate_user!
```

Comment:
Here you're turning off the requirement to be authenticated before you use this app. I think this is a mistake? !?

Reply:
i turned it back on